### ISSUE TITLE
Setting new parameters does not affect the current session

### ISSUE THREAD

**Issue by tsieger** [csn-le/wave_clus#28]
When setting new parameters using the `Set parameters` button, the new parameters do not take immediate effect, but apparently take effect after new data loading. 
It would be nice if some parameters, e.g. the parameters controlling the "force membership" feature (e.g. `par.template_sdnum`) or plotting parameters (e.g. `par.to_plot_std` or `par.max_spikes_plot`) could be set on the fly, such that the user could immediately see their effects for the clusters identified currently, without a need to reload the signal and re-apply operations leading to the given clusters.

**Comment by ferchaure** [comment]
You are right. At the beginning we did that because that was an easy way to implement the behaviour of the button (in the old wave_clus that button was useless). The idea was make a nice GUI like a form for that,  with some fields disable for each type of file; because, if you load a _spikes_ or _times_  the GUI will take the parameters inside them (detection and clustering parameters respectively).

But, we had a few problems with the implementation:
- Change the force parameters and fix classes could generate classes with different types of forced assignments and reduce the reproducibility of the sorting.
- We are trying to continue the style of the first wave_clus and keep the GUI as simple as possible 
- It require to change quite a lot of code and we need time for that.  

 However, I will try to work around this issue (at least for the plotting related parameters)

**Comment by ferchaure** [comment]
I added a line in the top of the "set parameters windows" explaining this behaviour
